plugins/org.eclipse.n4js.ide/src/org/eclipse/n4js/ide/xtext/server/build/ProjectStatePersister.java

```diff
@@ -154,19 +154,21 @@ public void close() {
 	 *
 	 * @param project
 	 *            the project
+	 * @param languageVersion
+	 *            the language version
 	 * @param state
 	 *            the state to be written
 	 */
-	public void writeProjectState(IProjectConfig project, ProjectState state) {
+	public void writeProjectState(IProjectConfig project, String languageVersion, ProjectState state) {
```

Comment:
Since we are writing the index data that was created by multiple languages, `serverVersion` could be the better term. Not sure though if it belongs into the config or if it should be an implementation detail of the persister.